### ISSUE TITLE
Photos and video in iOS 15 gallery are in a random order and most are not tappable

### ISSUE THREAD

**Issue by jgadsby** [GetStream/stream-chat-flutter#491]
**Describe the bug**
The title sums it up! Viewing the gallery preview in iOS 15 sorts all your images randomly, and tapping on most of them does nothing. 

**What package are you using? What version?**

**What platform is it about?**
- [ ] Android
- [X] iOS
- [ ] Web
- [ ] Windows
- [ ] MacOS
- [ ] Linux

**To Reproduce**
1. Run Stream on an iOS 15 device
2. Tap the attachment button
3. See the random order of images and videos / not tappable

**Comment by imtoori** [comment]
hey @jgadsby I just got to run the app on ios15 and everything is working fine
Can you confirm you're still having the issue?

**Comment by jgadsby** [comment]
@imtoori Yeah, I'm still seeing the issue. If you scroll past the first 10-20 photos (which are correctly in order) you'll begin to notice some videos from a long time ago, and they seem to get more jumbled as you scroll down more. If you don't have many photos it might not be noticeable.

**Comment by rickgrade3** [comment]
I encountered this issue too, Maybe MediaListView need to list image by "createdAt" instead "updatedAt".
https://github.com/CaiJingLong/flutter_photo_manager/issues/578#issuecomment-893017403
